crates/starknet/src/account.rs

```diff
@@ -256,17 +254,22 @@ mod tests {
     }
 
     fn setup() -> (Account, StarknetState) {
+        let mut state = StarknetState::default();
+        let fee_token_address = dummy_contract_address();
+
+        state.deploy_contract(fee_token_address, Felt::from(2)).unwrap();
```

Comment:
What is `Felt::from(2)`? Class hash? Why can it be any value? I see in another place in this PR it's introduced as `Felt::from(1)` and in another as `dummy_felt()`.

Reply:
the class hash in those tests is some arbitrary value. Fixed in [8217105](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/161/commits/8217105b9eb79f9cfeead030994db5b9386ca141) to use the erc20 contract class hash and dummy_felt in the other places